app/templates/custom-elements/remote-screen.html

```diff
@@ -153,29 +158,15 @@
 
           window.addEventListener("resize", this.onWindowResize);
 
-          // Detect whether this is a touchscreen device.
-          let isTouchScreen = false;
-          this.shadowRoot.addEventListener("touchend", () => {
-            isTouchScreen = true;
-          });
-          this.shadowRoot.addEventListener("click", () => {
-            if (isTouchScreen) {
-              this.shadowRoot.getElementById("mobile-keyboard-input").focus();
-            }
-          });
-
-          // On mobile, the keydown events function differently due to the OS
-          // attempting to autocomplete text. Instead of listening for keydown
-          // events, we listen for input events.
-          const mobileKeyboard = this.shadowRoot.getElementById(
-            "mobile-keyboard-input"
-          );
-          mobileKeyboard.addEventListener("input", () => {
-            // Force the autocomplete sequence to restart.
-            mobileKeyboard.blur();
-            mobileKeyboard.value = "";
-            mobileKeyboard.focus();
-          });
+          //needed for later keyboard PR
+          // Detect whether this is a touchscreen device .
+          // let isTouchScreen = false;
+          // this.shadowRoot.addEventListener("touchend", () => {
+          //   isTouchScreen = true;
+          // });
+          // this.shadowRoot.addEventListener("click", () => {
+          //   if (isTouchScreen) {
+          //     this.shadowRoot.getElementById("mobile-keyboard-input").focus();
```

Comment:
Can we delete this and pull it out of source history when we need it? We don't keep commented out code in the repo.

Reply:
Removed.